pandas/tests/scalar/timestamp/test_constructors.py

```diff
@@ -595,19 +595,16 @@ def test_out_of_bounds_string_consistency(self, arg):
         # GH 15829
         msg = "|".join(
             [
-                "Cannot cast 1-01-01 00:00:00 to unit='ns' without overflow",
-                "Out of bounds nanosecond timestamp: 1-01-01 00:00:00",
+                "Cannot cast 0001-01-01 00:00:00 to unit='ns' without overflow",
+                "Out of bounds nanosecond timestamp: 0001-01-01 00:00:00",
```

Comment:
To elaborate on my comment in the issue, my guess is that it used to be that different error messages were raised for the different input options in this test:
- For `"001-01-01"` the Timestamp constructor assumed nanosecond precision and so would not allow the construction of a nanosecond Timestamp with year equal one, raising the `"Out of bounds nanosecond timestamp: 0001-01-01 00:00:00"` message before one could try the `as_unit` call.  This necessitated the if-statement below, because it meant it was not possible to create a Timestamp object with `"001-01-01"` (i.e. previously line 608) without raising an error, so one could not query its `unit`.
- For `"0001-01-01"` a second precision Timestamp with year equal one was created, but when attempting to convert to nanosecond precision with `as_unit` the `"Cannot cast 0001-01-01 00:00:00 to unit='ns' without overflow"` message was raised.

Now that the Timestamp constructor appears to assume second precision for `"001-01-01"` as well, the `"Out of bounds nanosecond timestamp: 0001-01-01 00:00:00"` is no longer be raised for either of the two test cases and so the matching condition can be simplified to be the `"Cannot cast 0001-01-01 00:00:00 to unit='ns' without overflow"` message alone.

Reply:
thanks, I removed the check for the `Out of bounds` message